pepper-apis/src/main/java/org/broadinstitute/ddp/route/PatchFormAnswersRoute.java

```diff
@@ -618,6 +649,30 @@ private void checkAnswerConstraints(Response response, Answer answer) {
         }
     }
 
+    private void checkAndPrepareFile(Handle handle, Response response, ActivityInstanceDto instanceDto, FileAnswer answer) {
+        long participantId = instanceDto.getParticipantId();
+        long uploadId = answer.getValue().getUploadId();
+        var upload = handle.attach(FileUploadDao.class).findById(uploadId)
+                .orElseThrow(() -> new DDPException("Could not find file upload with id " + uploadId));
+        var checkResult = fileService.checkAndSetUploadStatus(handle, participantId, upload);
```

Comment:
Currently, we only check the file and set `UPLOADED` status if user actual attempts to PATCH with that file upload. If user (for whatever reason) requested a file upload, went ahead to finish the file upload, but doesn't PATCH it, then server won't ever look at that file upload. This means that file upload will continue to stay in `AUTHORIZED` status, and when we implement background job, the job will clean out and delete that file. I think this is fine because if user uploaded file but doesn't "assign" it to an answer, then it's effectively not "used", so cleaning it out should be fine.